anvil_consortium_manager/views.py

```diff
@@ -178,6 +190,123 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class AccountLink(LoginRequiredMixin, FormView):
+    login_url = settings.LOGIN_URL
+    template_name = "anvil_consortium_manager/account_form.html"
```

Comment:
This should have its own template, since we'll want to include some additional information about linking accounts for users.

Reply:
I added user_email_entry.html

Reply:
I added a new template, user_email_entry.html